spec/serializers/project_serializer_spec.rb

```diff
@@ -28,6 +28,18 @@
     end
   end
 
+  describe "#state" do
+    it "includes the state" do
+      expect(serializer.state).to eq project.state
+    end
+
+    it 'can filter by state' do
+      project.finished!
+      serializer = described_class.page({"state" => "finished"})
```

Comment:
Also does this actually test the filtering or just the matching factory data? (it probably does but would probably pass without the values passed to page...probably best to test for a matching filter with another non-matching factory in there?